handlers/chat.js

```diff
@@ -32,6 +32,17 @@ Dota2.Dota2Client.prototype._getChannelById = function(channel_id) {
     }
 }
 
+Dota2.Dota2Client.prototype._leaveChatChannelById = function(channelId, steamIdOfLeaver) {
+    var payload = new Dota2.schema.CMsgDOTALeaveChatChannel({
+        "channel_id": channelId
+    });
+    this.chatChannels = this.chatChannels.filter(item => item.channel_id.notEquals(channelId));
+    this.sendToGC(Dota2.schema.EDOTAGCMsg.k_EMsgGCLeaveChatChannel, payload);
+    if (this.debug) {
+        util.log(steamIdOfLeaver + " left channel " + channelId);
```

Comment:
This debug is somewhat counter-intuitive in the context of the function. This function makes the bot itself leave a channel so having a debug message saying someone else left might even be incorrect (take the case for example where this function would be called following a chat message or a join event). I propose changing the debug message to simply read `"Leaving channel" + channelId` and removing the parameter.